samtranslator/plugins/application/serverless_app_plugin.py

```diff
@@ -180,6 +180,9 @@ def _handle_get_application_request(self, app_id, semver, key, logical_id):
             warning_message = "{}. Unable to verify access to {}/{}.".format(e, app_id, semver)
             LOG.warning(warning_message)
             self._applications[key] = {"Unable to verify"}
+        except ClientError as e:
+            LOG.exception(e)
+            raise e
```

Comment:
Doesn't the exception get logged if this is raised and not caught? Any specific reason we added this here?

Reply:
I wondered the same thing -- I added this here to preserve the existing behavior.  I removed `LOG.exception` from `_sar_service_call` as throttles are not really exceptions for `GetCloudFormation`.  I can do a bit more digging to see if we need that extra log.